### PR TITLE
refactor(app): display file name from robot-server data

### PR DESCRIPTION
**Pull request by koji**
# Overview
display file name from robot-server data

close AUTH-571

Screenshot 2024-08-01 at 6 53 31 PM

### CODE REVIEW

File: app/src/organisms/Devices/ProtocolRun/ProtocolRunRunTimeParameters.tsx [hunk 1]
```diff
@@ -210,7 +210,7 @@ const StyledTableRowComponent = (
           <LegacyStyledText as="p">
             {parameter.type === 'csv_file'
               ? // TODO (nd, 07/17/2024): retrieve filename from parameter once backend is wired up
```
Comment (ncdiehl11) [inline app/src/organisms/Devices/ProtocolRun/ProtocolRunRunTimeParameters.tsx]: we can remove this comment I think

**Review by ncdiehl11 — COMMENTED** [review]
Is "name" a new field returned at the top level of the csv file runtime parameter?

**Comment by koji** [comment]
> Is "name" a new field returned at the top level of the csv file runtime parameter?

Yeah, robot-server returns the following format 

{
  id: string,
  name: string
}

**Review by ncdiehl11 — APPROVED** [review]
Looks good! Thank you Koji

**Pull request merged**